dts/arm/silabs/siwg917.dtsi

```diff
@@ -25,6 +25,19 @@
 			compatible = "arm,cortex-m4f";
 			reg = <0>;
 		};
+
+		power-states {
```

Comment:
The CPU should make use of compatible power states, at a minimum the `ps4_standby` state, but also the `ps4_sleep` state if it works out of the box.